.perfer/yoshi.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+const ROOT_DIR = process.cwd();
+const resolvePath = (...args) => path.resolve(ROOT_DIR, ...args);
+
+const componentsToBundle = require(resolvePath('../.wuf/components.json'));
+
+const components = Object.keys(componentsToBundle).reduce(
+  (accu, component) => ({
+    ...accu,
+    [component]: `${componentsToBundle[component].path.replace(
+      'src/',
+      '../../dist/es/src/'
+    )}/index`,
+  }),
+  {}
+);
+
+module.exports = {
+  entry: {
+    ...components,
+  },
+  externals: {
+    react: 'react',
+    'react-dom': 'reactDOM',
+    'prop-types': 'propTypes',
+    'react-is': 'react-is',
+  },
```

Comment:
Where does this come from 🤔 ?

Reply:
From `yoshi` probably (we use it to generate the bundles).

Without `externals`, this is the output:
```
| file                                                   | size      | gzipped  |
| ------------------------------------------------------ | --------- | -------- |
| .perfer/dist/statics/ActionsMenuLayout.bundle.min.js   | 37.054kb  | 7.826kb  |
| .perfer/dist/statics/Autocomplete.bundle.min.js        | 258.173kb | 73.771kb |
| .perfer/dist/statics/Avatar.bundle.min.js              | 53.776kb  | 17.616kb |
| .perfer/dist/statics/AvatarGroup.bundle.min.js         | 80.288kb  | 20.714kb |
| .perfer/dist/statics/Badge.bundle.min.js               | 15.188kb  | 5.162kb  |
| .perfer/dist/statics/Button.bundle.min.js              | 40.063kb  | 10.283kb |
| .perfer/dist/statics/CalendarCell.bundle.min.js        | 35.094kb  | 6.489kb  |
| .perfer/dist/statics/CalendarPopover.bundle.min.js     | 45.255kb  | 11.771kb |
| .perfer/dist/statics/Card.bundle.min.js                | 21.969kb  | 6.52kb   |
| .perfer/dist/statics/Checkbox.bundle.min.js            | 30.546kb  | 9.127kb  |
| .perfer/dist/statics/CheckboxGroup.bundle.min.js       | 14.492kb  | 5.136kb  |
| .perfer/dist/statics/ColorPicker.bundle.min.js         | 250.156kb | 71.826kb |
| .perfer/dist/statics/CopyUrlButton.bundle.min.js       | 263.937kb | 75.428kb |
| .perfer/dist/statics/Counter.bundle.min.js             | 247.129kb | 71.356kb |
| .perfer/dist/statics/Dialog.bundle.min.js              | 42.475kb  | 12.513kb |
| .perfer/dist/statics/Divider.bundle.min.js             | 12.148kb  | 4.479kb  |
| .perfer/dist/statics/DotNavigation.bundle.min.js       | 32.271kb  | 9.922kb  |
| .perfer/dist/statics/Dropdown.bundle.min.js            | 334.843kb | 84.389kb |
| .perfer/dist/statics/Event.bundle.min.js               | 39.067kb  | 10.293kb |
| .perfer/dist/statics/FloatingDropdown.bundle.min.js    | 273.974kb | 76.581kb |
| .perfer/dist/statics/Grid.bundle.min.js                | 18.214kb  | 6.291kb  |
| .perfer/dist/statics/IconButton.bundle.min.js          | 28.453kb  | 8.895kb  |
| .perfer/dist/statics/IconToggle.bundle.min.js          | 25.703kb  | 8.249kb  |
| .perfer/dist/statics/Image.bundle.min.js               | 48.003kb  | 14.557kb |
| .perfer/dist/statics/Input.bundle.min.js               | 25.129kb  | 7.85kb   |
| .perfer/dist/statics/LikeButton.bundle.min.js          | 33.439kb  | 9.234kb  |
| .perfer/dist/statics/NewCard.bundle.min.js             | 13.878kb  | 5.055kb  |
| .perfer/dist/statics/OverlappingCard.bundle.min.js     | 28.813kb  | 7.151kb  |
| .perfer/dist/statics/Pagination.bundle.min.js          | 43.577kb  | 12.276kb |
| .perfer/dist/statics/Picker.bundle.min.js              | 39.848kb  | 10.521kb |
| .perfer/dist/statics/ProgressBar.bundle.min.js         | 24.303kb  | 7.723kb  |
| .perfer/dist/statics/RadioButton.bundle.min.js         | 28.707kb  | 8.731kb  |
| .perfer/dist/statics/RadioButtonGroup.bundle.min.js    | 14.821kb  | 5.221kb  |
| .perfer/dist/statics/Ratings.bundle.min.js             | 30.72kb   | 9.336kb  |
| .perfer/dist/statics/SectionNotification.bundle.min.js | 52.087kb  | 12.061kb |
| .perfer/dist/statics/ShareButton.bundle.min.js         | 32.628kb  | 9.746kb  |
| .perfer/dist/statics/SocialBar.bundle.min.js           | 252.046kb | 72.394kb |
| .perfer/dist/statics/Spinner.bundle.min.js             | 13.629kb  | 5.003kb  |
| .perfer/dist/statics/StatesButton.bundle.min.js        | 44.508kb  | 11.503kb |
| .perfer/dist/statics/StripCard.bundle.min.js           | 15.141kb  | 5.349kb  |
| .perfer/dist/statics/Tabs.bundle.min.js                | 188.388kb | 55.357kb |
| .perfer/dist/statics/Tags.bundle.min.js                | 32.479kb  | 9.785kb  |
| .perfer/dist/statics/Text.bundle.min.js                | 15.563kb  | 5.144kb  |
| .perfer/dist/statics/TextArea.bundle.min.js            | 245.398kb | 70.94kb  |
| .perfer/dist/statics/TextButton.bundle.min.js          | 29.857kb  | 9.14kb   |
| .perfer/dist/statics/TextField.bundle.min.js           | 261.909kb | 74.507kb |
| .perfer/dist/statics/Toast.bundle.min.js               | 34.256kb  | 10.173kb |
| .perfer/dist/statics/ToggleSwitch.bundle.min.js        | 26.384kb  | 7.835kb  |
| .perfer/dist/statics/Tooltip.bundle.min.js             | 238.666kb | 69.519kb |
```

With `externals`:
```
| file                                                   | size      | gzipped  |
| ------------------------------------------------------ | --------- | -------- |
| .perfer/dist/statics/ActionsMenuLayout.bundle.min.js   | 30.072kb  | 5.308kb  |
| .perfer/dist/statics/Autocomplete.bundle.min.js        | 131.779kb | 34.263kb |
| .perfer/dist/statics/Avatar.bundle.min.js              | 46.793kb  | 15.08kb  |
| .perfer/dist/statics/AvatarGroup.bundle.min.js         | 73.304kb  | 18.185kb |
| .perfer/dist/statics/Badge.bundle.min.js               | 8.207kb   | 2.64kb   |
| .perfer/dist/statics/Button.bundle.min.js              | 33.078kb  | 7.878kb  |
| .perfer/dist/statics/CalendarCell.bundle.min.js        | 28.112kb  | 3.966kb  |
| .perfer/dist/statics/CalendarPopover.bundle.min.js     | 38.271kb  | 9.331kb  |
| .perfer/dist/statics/Card.bundle.min.js                | 14.987kb  | 4.022kb  |
| .perfer/dist/statics/Checkbox.bundle.min.js            | 23.563kb  | 6.665kb  |
| .perfer/dist/statics/CheckboxGroup.bundle.min.js       | 7.511kb   | 2.641kb  |
| .perfer/dist/statics/ColorPicker.bundle.min.js         | 123.763kb | 32.316kb |
| .perfer/dist/statics/CopyUrlButton.bundle.min.js       | 137.541kb | 35.822kb |
| .perfer/dist/statics/Counter.bundle.min.js             | 120.735kb | 31.846kb |
| .perfer/dist/statics/Dialog.bundle.min.js              | 35.49kb   | 10.09kb  |
| .perfer/dist/statics/Divider.bundle.min.js             | 5.167kb   | 1.963kb  |
| .perfer/dist/statics/DotNavigation.bundle.min.js       | 25.289kb  | 7.47kb   |
| .perfer/dist/statics/Dropdown.bundle.min.js            | 208.447kb | 44.803kb |
| .perfer/dist/statics/Event.bundle.min.js               | 32.083kb  | 7.859kb  |
| .perfer/dist/statics/FloatingDropdown.bundle.min.js    | 147.578kb | 37.061kb |
| .perfer/dist/statics/Grid.bundle.min.js                | 11.232kb  | 3.78kb   |
| .perfer/dist/statics/IconButton.bundle.min.js          | 21.469kb  | 6.487kb  |
| .perfer/dist/statics/IconToggle.bundle.min.js          | 18.72kb   | 5.798kb  |
| .perfer/dist/statics/Image.bundle.min.js               | 41.021kb  | 12.131kb |
| .perfer/dist/statics/Input.bundle.min.js               | 18.147kb  | 5.397kb  |
| .perfer/dist/statics/LikeButton.bundle.min.js          | 26.456kb  | 6.778kb  |
| .perfer/dist/statics/NewCard.bundle.min.js             | 6.896kb   | 2.559kb  |
| .perfer/dist/statics/OverlappingCard.bundle.min.js     | 21.832kb  | 4.641kb  |
| .perfer/dist/statics/Pagination.bundle.min.js          | 36.596kb  | 9.81kb   |
| .perfer/dist/statics/Picker.bundle.min.js              | 32.863kb  | 8.094kb  |
| .perfer/dist/statics/ProgressBar.bundle.min.js         | 17.321kb  | 5.259kb  |
| .perfer/dist/statics/RadioButton.bundle.min.js         | 21.726kb  | 6.265kb  |
| .perfer/dist/statics/RadioButtonGroup.bundle.min.js    | 7.84kb    | 2.726kb  |
| .perfer/dist/statics/Ratings.bundle.min.js             | 23.738kb  | 6.889kb  |
| .perfer/dist/statics/SectionNotification.bundle.min.js | 45.103kb  | 9.601kb  |
| .perfer/dist/statics/ShareButton.bundle.min.js         | 25.644kb  | 7.316kb  |
| .perfer/dist/statics/SocialBar.bundle.min.js           | 125.65kb  | 32.821kb |
| .perfer/dist/statics/Spinner.bundle.min.js             | 6.647kb   | 2.49kb   |
| .perfer/dist/statics/StatesButton.bundle.min.js        | 37.523kb  | 9.079kb  |
| .perfer/dist/statics/StripCard.bundle.min.js           | 8.159kb   | 2.859kb  |
| .perfer/dist/statics/Tabs.bundle.min.js                | 62.003kb  | 16.207kb |
| .perfer/dist/statics/Tags.bundle.min.js                | 24.768kb  | 7.031kb  |
| .perfer/dist/statics/Text.bundle.min.js                | 8.581kb   | 2.63kb   |
| .perfer/dist/statics/TextArea.bundle.min.js            | 118.999kb | 31.452kb |
| .perfer/dist/statics/TextButton.bundle.min.js          | 22.873kb  | 6.707kb  |
| .perfer/dist/statics/TextField.bundle.min.js           | 135.514kb | 34.925kb |
| .perfer/dist/statics/Toast.bundle.min.js               | 27.271kb  | 7.755kb  |
| .perfer/dist/statics/ToggleSwitch.bundle.min.js        | 19.402kb  | 5.378kb  |
| .perfer/dist/statics/Tooltip.bundle.min.js             | 112.272kb | 30.011kb |
```

Reply:
So I guess this list should be aligned with the TPA's external resources 🤔